presto-hive/src/main/java/io/prestosql/plugin/hive/HiveTableProperties.java

```diff
@@ -50,6 +50,8 @@
     public static final String ORC_BLOOM_FILTER_COLUMNS = "orc_bloom_filter_columns";
     public static final String ORC_BLOOM_FILTER_FPP = "orc_bloom_filter_fpp";
     public static final String AVRO_SCHEMA_URL = "avro_schema_url";
+    public static final String TEXT_SKIP_HEADER_COUNT = "skip_header_count";
+    public static final String TEXT_SKIP_FOOTER_COUNT = "skip_footer_count";
```

Comment:
... then this would be something like:

```suggestion
    public static final String TEXTFILE_SKIP_FOOTER_LINE_COUNT = "textfile_skip_footer_line_count";
```